src/uct/api/v2/uct_v2.h

```diff
@@ -516,6 +517,16 @@ typedef struct uct_md_mem_attach_params {
 extern const char *uct_ep_operation_names[];
 
 
+/**
+ * @ingroup UCS_RESOURCE
+ * @brief Reachability scope
+ */
+typedef enum uct_iface_reachability_scope {
+    UCT_IFACE_REACHABILITY_INTRA_NODE, /**< Intra node scope */
+    UCT_IFACE_REACHABILITY_INTER_NODE, /**< Inter node scope */
```

Comment:
what if it's same node but different device?

Reply:
As we already changed the API several times, maybe should discuss this issue @yosefe , @gleon99, a
and define API which is acceptible for all

Reply:
talked w @gleon99 , the param can remain const/in since the scope is input and reachable indication is function return value